src/app/zones/views/ZoneEgressListView.vue

```diff
@@ -144,25 +153,32 @@ function transformToTableData(zoneEgressOverviews: ZoneEgressOverview[]): ZoneEg
     const status = getItemStatusFromInsight(entity.zoneEgressInsight ?? {})
 
     return {
-      entity,
+      id,
       detailViewRoute,
+      name,
       status,
     }
   })
 }
+</script>
```

Comment:
I think I commented on this before but I can no longer see the comment on GH. Is there any reason not to just do this inline?

Reply:
One of the other threads is here: https://github.com/kumahq/kuma-gui/pull/1209#discussion_r1268155151

> I’m not doing this. There is no point in reshaping all this data in the template when we can do it how we do it right now.

Reply:
Half the point of this stream of work is to reduce all the complexity in just rendering some JSON to a list. I posted the other day on Slack about the "Small functions considered Harmful".

If there is something here that is overly-complex to do inline please point it out to me.

Reply:
It doesn’t need changing. It’s fine. A function transforming data between formats isn’t increasing the complexity of a component.

I really, really, really, really don’t want to change this. There is no need and I personally think the suggested alternative is worse (something which I’ve pointed out numerous times by now).